torch/__init__.py

```diff
@@ -270,22 +270,30 @@ def set_default_tensor_type(t):
 
 
 def set_default_dtype(d):
-    r"""Sets the default floating point dtype to :attr:`d`. This type will be
-    used as default floating point type for type inference in
-    :func:`torch.tensor`.
+    r"""Sets the default floating point dtype to :attr:`d`.
+    This dtype is:
+    1. The inferred dtype for python floats in :func:`torch.tensor`.
+    2. Used to determine the default complex dtype. The default complex dtype is set to
```

Comment:
same issue with 1 -- this should say something like "the inferred dtype for python complex numbers"

Reply:
does this sound better?

```
Used to infer dtype for python complex numbers. The default complex dtype is set to
 ``torch.complex128`` if default floating point dtype is ``torch.float64``, otherwise it's set to ``torch.complex64``
```

Reply:
lgtm!